src/test/java/com/wikia/webdriver/testcases/mercurytests/CuratedMainPageTests.java

```diff
@@ -143,13 +150,40 @@ public void MercuryCuratedMainPageTests_002_CheckElementsInvisibility() {
   @Test(groups = {"MercuryCuratedMainPageTests_003", "MercuryCuratedMainPageTests", "Mercury"})
   public void MercuryCuratedMainPageTests_003_CheckElementsForNoTrendingArticles() {
     BasePageObject base = new BasePageObject(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_NTA_CC);
     base.openMercuryArticleByName(wikiURL, MercuryArticles.NTACC_MAIN_PAGE);
     CuratedPageObject cc = new CuratedPageObject(driver);
 
     PageObjectLogging.logWarning("Check", "Element invisibility");
-    PageObjectLogging
-        .log(PageElements.TRENDING_ARTICLES.name, MercuryMessages.INVISIBLE_MSG,
-             MercuryMessages.VISIBLE_MSG, !cc.isTrendingArticlesVisible());
+    PageObjectLogging.log(PageElements.TRENDING_ARTICLES.name, MercuryMessages.INVISIBLE_MSG,
+                          MercuryMessages.VISIBLE_MSG, !cc.isTrendingArticlesVisible());
+
+    PageObjectLogging.logWarning("Check", "Elements visibility");
+    PageObjectLogging.log(PageElements.TOP_LEADERBOARD.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isMobileTopLeaderboardVisible());
+    PageObjectLogging.log(PageElements.ARTICLE_TITLE.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isArticleTitleVisible());
+    PageObjectLogging.log(PageElements.WIKIA_STATS.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isWikiaStatsContainerVisible());
+    PageObjectLogging.log(PageElements.FEATURED_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isFeaturedContentVisible());
+    PageObjectLogging.log(PageElements.CURATED_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isCuratedContentVisible());
+    PageObjectLogging.log(PageElements.IN_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isMobileInContentVisible());
+    PageObjectLogging.log(PageElements.TRENDING_VIDEOS.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isTrendingVideosVisible());
+    PageObjectLogging.log(PageElements.PREFOOTER.name, MercuryMessages.VISIBLE_MSG,
+                          MercuryMessages.INVISIBLE_MSG, cc.isMobilePrefooterVisible());
+    }
+
+  // CCT04
+  @Test(groups = {"MercuryCuratedMainPageTests_004", "MercuryCuratedMainPageTests", "Mercury"})
+  public void MercuryCuratedMainPageTests_004_CheckElementsVisibilityExceptTrendingVideos() {
+    BasePageObject base = new BasePageObject(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_NTV_CC);
+    base.openMercuryArticleByName(wikiURL, MercuryArticles.NTVCC_MAIN_PAGE);
+    CuratedPageObject cc = new CuratedPageObject(driver);
```

Comment:
why you create base object?
You could do:
```java
page = new CuratedPageObject(driver);
page.openMercuryArticleByName...
```
This way less variables would be created and test would be shorter a bit